api_tests/lib/bitcoind_instance.ts

```diff
@@ -78,4 +64,31 @@ export class BitcoindInstance implements LedgerInstance {
     public getUsernamePassword() {
         return { username: this.username, password: this.password };
     }
+
+    public getDataDir() {
+        return this.dbDir.name;
+    }
+
+    private writeLogFile() {
+        const output = `regtest=1
+server=1
+printtoconsole=1
+bind=0.0.0.0:${this.p2pPort}
+rpcbind=0.0.0.0:${this.rpcPort}
+rpcallowip=0.0.0.0/0
+nodebug=1
+rest=1
+acceptnonstdtxn=0
+zmqpubrawblock=tcp://127.0.0.1:28332
+zmqpubrawtx=tcp://127.0.0.1:28333
+`;
+        const config = this.dbDir.name + "/bitcoin.conf";
+
+        fs.writeFile(config, output, function(err: any) {
```

Comment:
Any reason for deviating from using async/await here?

Reply:
Until these comments I had paid no attention, and had no idea, that there were async and sync versions of all the file system commands.

Reply:
> Any reason for deviating from using async/await here?

No reason except that this works so why promisify-ing? This clearly something obvious to you @thomaseizinger but not us.